.zenodo.json

```diff
@@ -0,0 +1,34 @@
+{
+  "license": {
+  "id": "MIT"
+  }, 
+  "notes": "When referencing this package, please cite both the package DOI and the Apex Coordinates journal article: Emmert, J. T., A. D. Richmond, and D. P. Drob (2010), A computationally compact representation of Magnetic-Apex and Quasi-Dipole coordinates with smooth base vectors, J. Geophys. Res., 115(A8), A08322, doi:10.1029/2010JA015326.",
+  "references": [
+    "Emmert, J. T., A. D. Richmond, and D. P. Drob (2010), A computationally compact representation of Magnetic-Apex and Quasi-Dipole coordinates with smooth base vectors, J. Geophys. Res., 115(A8), A08322, doi:10.1029/2010JA015326.",
+    "Richmond, A. D. (1995), Ionospheric Electrodynamics Using Magnetic Apex Coordinates, Journal of geomagnetism and geoelectricity, 47(2), 191–212, doi:10.5636/jgg.47.191."
+  ], 
+  "keywords": [
+    "Magnetic Apex Coordinates",
+    "Quasi Dipole Coordinates",
+    "MLT", 
+    "Magnetic Local Time", 
+    "Conversion", 
+    "Coordinate Conversion", 
+    "Converting", 
+    "Ionosphere", 
+    "Magnetic Field", 
+    "Space Physics", 
+    "Heliophysics"
+  ], 
+  "creators": [
+    {
+      "orcid": "0000-0001-8875-9326", 
+      "affiliation": "Naval Research Laboratory", 
+      "name": "Angeline G. Burrell"
+    }, 
+    {
+      "orcid": "0000-0002-8043-0953", 
+      "name": "Christer van der Meeren"
+    }
```

Comment:
We need to add the rest of the authors from the Authors.md file here.  The order should be:
1) Christer
2) Karl
3) Angeline
4) Greg
5) Ashton
6) Achim

Reply:
Ok wasn't sure, thanks for the feedback

Reply:
@sputnik-a is this you?

https://orcid.org/0000-0003-4227-8687

Reply:
Nope that's not him. I don't think he has orcid

Reply:
No, that's not me. Here is my Orc-ID: https://orcid.org/0000-0001-7955-4441